tools/applicationinsights-web-snippet/src/snippet.ts

```diff
@@ -194,12 +207,31 @@ declare var cfg:ISnippetConfig;
                 });
                 // let message = "Load Version 2 SDK instead to support IE"; // where to report this error?
             }
+
+            if (cfg.cr != false){
```

Comment:
This should be `!==` so that it's an exact match, otherwise the value of `cfg.cr` will get coerced to get compared against `false`